support-workers/src/main/scala/com/gu/emailservices/DigitalPackEmailFields.scala

```diff
@@ -211,12 +211,6 @@ class DigitalPackEmailFields(
       state: SendThankYouEmailDigitalSubscriptionGiftPurchaseState,
   )(implicit ec: ExecutionContext) = {
     import state._
-
```

Comment:
I noticed that the applied promotion is never used by the email fields generator so I removed the code related to it.